compiler/package.go

```diff
@@ -120,7 +120,20 @@ func (pi packageImporter) Import(path string) (*types.Package, error) {
 	return pi.importContext.Packages[a.ImportPath], nil
 }
 
-func Compile(importPath string, files []*ast.File, fileSet *token.FileSet, importContext *ImportContext, minify bool) (*Archive, error) {
+func Compile(importPath string, files []*ast.File, fileSet *token.FileSet, importContext *ImportContext, minify bool) (_ *Archive, comError error) {
```

Comment:
Consider modifying `err := recover()` to `e := recover()` and renaming `comError` to `err`, since that might look cleaner. `comError` only makes sense in the context of the defer, otherwise it is just the error that Compile returns, so `err` is a simple and clear name.